android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -448,9 +445,26 @@ protected void onScrollChanged(int l, int t, int oldl, int oldt)
 				KrollDict data = new KrollDict();
 				getProxy().fireEvent(TiC.EVENT_DRAGSTART, data);
 			}
+
+			double contentWidth = 0;
```

Comment:
Maybe make a method that returns a `contentData` `KrollDict` so we don't have duplicate code?

Reply:
good idea! Will change that